lib/frame.rb

```diff
@@ -0,0 +1,26 @@
+
+class Frame
+	def initialize
+		@score = []
+	end
+
+	def score
+		@score << first_roll
+
+		if first_roll < 10
+			@score << second_roll
+		end
+	end
+
+	def roll
+		first_roll
+	end
+
+	def first_roll
+		rand(10)
+	end
+
+	def second_roll
+		rand(10)
+	end
```

Comment:
I like this way of doing it - I didn't even think to make methods for the rolls, I was sitting on manually inputting. Well done, this is an easier way to see if your score code is working as intended.